tsconfig.json

```diff
@@ -1,13 +1,16 @@
 {
   "compilerOptions": {
     "module": "commonjs",
+    "target": "es2018",
```

Comment:
Added some properties to align with the `tsconfig.json` added [here](https://github.com/graphql/graphql-js/pull/2609).

Reply:
Merging as is but can you please research if `es2018` already includes async iteratable?
Also can you please research what the best way tie it to Node v10?